[EDU-6046] Create Azion Lib docs in English (#1479)
aziontech/docs: src/content/docs/en/pages/devtools/azion-lib/usage/config.mdx

---
title: Azion `Config` module
description: >-

>   Configure and validate options for Edge Application using the Config module.

Reviewer: ```suggestion
  Configure and validate options for Edge Applications using the Config module.
```

permalink: /documentation/products/azion-lib/config/
meta_tags: >-
  Azion Lib, edge computing, edge applications, Azion products
namespace: documentation_azion_lib_config
---

import LinkButton from 'azion-webkit/linkbutton'

The **Config** module provides a function to configure and validate options for **Edge Application**. It supports various configurations, including domain settings, origin settings, cache settings, rules, network lists, and purge operations.

> 
> ## Functions

Reviewer 2: ```suggestion

---

## Functions
```

### `defineConfig`

> This function configures and validates the options for the Azion Edge Application. It receives a configuration object which contains the setting for domains, cache, rules, and more.

Reviewer 2: ```suggestion
This function configures and validates the options for Azion Edge Application. It receives a configuration object that contains the settings for domains, cache, rules, and more.
```
